handson/Ansible/Ansible Lessons.md

```diff
@@ -2,8 +2,12 @@
 
 This directory contains the lesson plans for the Ansible  part of the bootcamp. Each lesson has it's own directory, which contains the description of the tasks to complete during the lesson, in markdown format, as well as supporting files such as inventory or playboook stubs, _etc_.
 
-In order to do the course, go through the lesson description step by step, completing and ticking off tasks as you go. If there are questions, search for or start a topic on the forum linked to that lesson.
```

Comment:
Yes, we want people to go to the forum to discuss the course, but when there are actual errors in the content _in this repo_ then it's best to deal with those using the github tickets, don't you think ?